openzwave-sys/openzwave-c/manager.h

```diff
@@ -68,26 +68,28 @@ GET_VALUE_FUNC(list_items, void ** value, const RustStringVecCreator);
 GET_VALUE_FUNC(list_values, void ** value, const RustI32VecCreator);
 
 #define GET_NODE_FUNC(name, return_type, ...) \
-  return_type manager_node_ ## name (Manager * manager, uint32 const home_id, uint8 const node_id, ##__VA_ARGS__)
+  return_type manager_node_ ## name (Manager * manager, uint32 home_id, uint8 node_id, ##__VA_ARGS__)
```

Comment:
1. because the const is not needed here. uint8 is copied, uint32 is copied
2. I guess we don't.